astropy/units/physical/__init__.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Defines the physical types that correspond to different units.
+
+The classes and functions defined here are also available in
+(and should be used through) the `astropy.units` namespace.
+"""
+
+from . import core
+from .core import *
+from .core import _name_physical_mapping  # sphinx-astropy needs this when building docs
+from .definitions import *
+
+__all__ = list(core.__all__)
+
+
+def __getattr__(name):
```

Comment:
My sense would be to just move `__getattr__` and `__dir__` up from `core` (and thus import `_attrname_physical_mapping`)